roles/django/templates/local_settings.py

```diff
@@ -0,0 +1,95 @@
+# Cryptographic key for signing secrets. Keep the production key hidden!
```

Comment:
Has this changed at all, or just moved to a new location?

Reply:
it did change; i grouped things into sections and did some formatting. there are also more things that are variables in this version vs. the old one.